### ISSUE TITLE
Error: Maximum execution time of 30 seconds exceeded

### ISSUE THREAD

**Issue by ondrejhlavacek** [keboola/syrup-queue#84]
```
message: Error: Maximum execution time of 30 seconds exceeded
level: 0
channel: app
datetime: {
date: 2018-12-19 10:10:55
}
app: queue
priority: CRITICAL
file: /www/syrup-router/components/queue/releases/1544750711/vendor/aws/aws-sdk-php/src/Api/Parser/PayloadParserTrait.php
line: 17
pid: 16361
trace: #0 {main}

exceptionId: queue-9cdd1f69086ccc38f5d15b4a91a6d001
}
```

**Comment by ondrejhlavacek** [comment]
> ked pojdes na project power stranku a mas tam vela dat, tak to nestihne za 30s preratat (agreguje sa to v php) ... sere nas to, ale je to "known issue"
> ak si spominas, sam si tam nejako zmensoval ten limit aby sa to tahalo len za 3 mesiace ... no aj tak sa to obcas stane

https://keboola.slack.com/archives/C13CP6YMV/p1545221607003300

**Comment by MiroCillik** [comment]
No teraz sa to stava nejak casto, ci nastal ten "3 mesacny" cas zrovna? :D

**Comment by MiroCillik** [comment]
Nedalo by sa to nejak pre-agregovat uz ked sa to uklada?

**Comment by MiroCillik** [comment]
Este sem odlozim https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/bp-gsi-aggregation.html

**Comment by MiroCillik** [comment]
Cez Dynamo Streamy by sa to dalo spravit, tak, ze pri kazdom update tabulky sa to cez stream zagreguje do druhej tabulky.
https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.html

**Comment by ujovlado** [comment]
Jo, to vieme. Myslim ale, ze bola skor idea to cele dat do Elasticu a DynamoDB pouzit  len na ten to prvotne zbieranie.

**Comment by MiroCillik** [comment]
Nevies, kde ja ta povodna issue k tomu? V Elasticu by sa nad tym dali robit rozne searche.

**Comment by ujovlado** [comment]
Tu je o tom viac: https://github.com/keboola/syrup-queue/issues/76

**Comment by Halama** [comment]
A nešlo by ten timeout alespoň zvednout?

**Comment by yustme** [comment]
Tenhle týden už to mám po 3. Stává se to pravidelně na projektu "Rohlík 2.0"

**Comment by Halama** [comment]
Opět https://keboola.slack.com/archives/C02CGK5CL/p1556180238006000
![image](https://user-images.githubusercontent.com/903531/56721083-6ddabe80-6744-11e9-8ca9-827c22985360.png)

**Comment by Halama** [comment]
Opět 2x https://keboola.slack.com/archives/C02CGK5CL/p1556228437000600

**Comment by Halama** [comment]
https://keboola.slack.com/archives/C02CGK5CL/p1556244152000900

**Comment by ondrejhlavacek** [comment]
![image](https://user-images.githubusercontent.com/497675/59224329-6d03cc00-8bce-11e9-8436-454754a07aac.png)

https://keboola.slack.com/archives/C02CGK5CL/p1560188506032800

**Comment by pivnicek** [comment]
myslim ze dnes dostal opet https://keboola.slack.com/archives/C02CGK5CL/p1560772177017700

**Comment by pivnicek** [comment]
This is happening more frequently last 2 days

**Comment by odinuv** [comment]
3x times on amall https://keboola.slack.com/archives/C02CGK5CL/p1562079074100100
